patches/v8/0019-Modify-where-to-look-for-fast_float-and-simdutf.patch

```diff
@@ -5,24 +5,13 @@ Subject: Modify where to look for fast_float and simdutf.
 
 Similar to fp16, these dependencies now needs to be downloaded by bazel.
 
+Signed-off-by: James M Snell <jsnell@cloudflare.com>
+
 diff --git a/BUILD.bazel b/BUILD.bazel
-index 1f3bb2ab91092e8235f01c1474da63776e6cb13e..37d1a068da6bac22966743f9617d7961df084632 100644
+index a73cfb6e5394041ad502d4bb7ddc83324dc34871..b2e8c9502294b17bd7c20dc463b0bffc028b34b9 100644
 --- a/BUILD.bazel
 +++ b/BUILD.bazel
-@@ -4346,12 +4346,6 @@ cc_library(
-     ],
- )
- 
--cc_library(
--  name = "simdutf",
```

Comment:
Why did we remove this? I thought this was required in order to differentiate our simdutf dependency with v8's

Reply:
V8's build includes this already and there was a merge conflict... this is what is in the v8 build now:

```
cc_library(
  name = "simdutf",
  srcs = ["third_party/simdutf/simdutf.cpp"],
  hdrs = ["third_party/simdutf/simdutf.h"],
  copts = select({
        "@v8//bazel/config:is_clang": ["-std=c++20"],
        "@v8//bazel/config:is_gcc": ["-std=gnu++2a"],
        "@v8//bazel/config:is_windows": ["/std:c++20"],
        "//conditions:default": [],
    }),
)
```

Reply:
ah... I think I just actually understood this patch lol. It *needs* to be removed from the v8 build file. Sigh. Moving slow today